docs/file-formats/json-project/ontologies.md

```diff
@@ -264,7 +264,7 @@ These three are related as follows:
 | isPartOf                    | (resourceclass)                                                     | Searchbox                            |
 | seqnum                      | IntValue                                                            | Spinbox, <br>SimpleText              |
 | isSequenceOf                | (AudioRepresentation,<br>MovingImageRepresentation,<br>or subclass) | Searchbox                            |
-| hasSequenceBounds           | IntervalValue                                                       | Interval                             |
+| hasSequenceBounds           | IntervalValue                                                       | Interval, <br>SimpleText             |
```

Comment:
I'm having a hard time imagine an IntervalValue working with a SimpleText text input. Is that supported by the APP? And if not, should TOOLS be able to do anything that the APP doesn't support?

Reply:
Good point. It was in the docs (further down), so I added it here to make it consistent.

But now that I think about it, it makes absolutely no sense. I removed support for it.